src/didactWebView.ts

```diff
@@ -266,26 +266,37 @@ export class DidactWebviewPanel {
 		// And the uri we use to load this script in the webview
 		const scriptUri = scriptPathOnDisk.with({ scheme: 'vscode-resource' });
 
-		let cssFile = 'webviewslim.css';
-		if (this.isAsciiDoc) {
-			cssFile = 'asciidoctor.css';
-		}
+		const bulmaCssHtml = `<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">`;
+
+		const adCssPathOnDisk = vscode.Uri.file(
+			path.join(this._extensionPath, 'media', 'asciidoctor.css')
+		);
+		const adCssUri = adCssPathOnDisk.with({ scheme: 'vscode-resource' });
+		const adUriHtml = `<link rel="stylesheet" href="${adCssUri}"/>`;
```

Comment:
Maybe we can put these lines speciifc to Asciidoc inside the if block L.284 ?
it will avoid to compute paths when Markdown is used.
I think it will be also easier to understand which part is related to AsciiDoc and which is related to marksdown.